frontend/src/__tests__/cypress/cypress/tests/mocked/projects/workbench.cy.ts

```diff
@@ -399,10 +400,72 @@ describe('Workbench page', () => {
     });
 
     cy.get('@editWorkbench.all').then((interceptions) => {
-      expect(interceptions).to.have.length(2); // 1 dry run request and 1 actaul request
+      expect(interceptions).to.have.length(2); // 1 dry run request and 1 actual request
     });
   });
 
+  it('Handle deleted notebook sizes in workbenches table', () => {
+    initIntercepts({});
+    cy.interceptOdh(
+      'GET /api/config',
+      mockDashboardConfig({
+        notebookSizes: [
```

Comment:
Could this be added to initIntercepts and just configured with these values? 

I imagine the test is not finding any mock today for this and soft erroring out... the mock dashboard config seems to handle no data -- so it shouldn't impact things 🤔 but it will allow for wider usage by other tests down the road.

Also, you don't need to make the mock so much -- you're just looking to set it to something else for the sake of this test... verbose test data doesn't necessarily help test it more.